library/src/main/java/com/timehop/stickyheadersrecyclerview/HeaderPositionCalculator.java

```diff
@@ -19,6 +19,9 @@
   private final OrientationProvider mOrientationProvider;
   private final HeaderProvider mHeaderProvider;
   private final DimensionCalculator mDimensionCalculator;
+  
+  private final Rect bufferRect1 = new Rect();
+  private final Rect bufferRect2 = new Rect();
```

Comment:
Previously they were local variables and called `headerMargins` and `nextHeaderMargins`, maybe we can just call them `mHeaderRect` and `mNextHeaderRect` or something